### DISCUSSION TITLE
Using relationship's `_id` in bindings - DONE

### DISCUSSION THREAD

**Discussion by mjashanks** [Budibase/budibase#2617] [Ideas]
We should be able to have access to the relationship `_id` field, when in bindings, like `{{ Repeater.Table.RelatedColumn._id }}`.

Currently, all we can do is `{{ Repeater.Table.RelatedColumn }}`, which displays the related row's display name.

My primary use of this, which I have run into a number of times is - Navigate To related record. 

e.g. 
- One `Company` has many `Contacts`
- from the `/contact/:id` detail, define action on the "View Company" button, to navigate to 

`/company/{{ Repeater.Contact.Company._id }}`

Maybe we could work this out so that that `{[ Repeater.Contact.Company }}` still renders the display name, thus not breaking existing apps. e.g. can we override handlebars' `toString()` to look for a `{ _displayName }` field?

**Comment by mjashanks** [comment]
Another related use of this: https://github.com/Budibase/budibase/discussions/2800

Check out the video below - when creating/updating a record, I want to copy a relationship from another entity, but cannot, because the other entity's relationship is loaded by display name only.

https://user-images.githubusercontent.com/3524181/135425976-44f83351-93b3-46b5-a987-e812774f8e4c.mp4

**Comment by mjashanks** [comment]
Whilst not strictly related to `_id`, this discussion demonstrates the need to be able to access the underlying relationship array, with bindings: https://github.com/Budibase/budibase/discussions/3109

**Reply by gvlx** [reply]
#3365 solved this. And the dual level on the URL too: /path/:id/:parent_id
